addons/mail/models/mail_tracking_value.py

```diff
@@ -46,7 +46,7 @@ def has_field_access(tracking):
             if not tracking.field_id:
                 return env.is_system()
             model_field = env[tracking.field_id.model]._fields.get(tracking.field_id.name)
-            return model_field.is_accessible(env)
+            return model_field.is_accessible(env) if model_field else env.is_system()
```

Comment:
```suggestion
            return model_field.is_accessible(env) if model_field else False
```

Since there is no `model_fiels` simply return False; otherwise, it shows a 'None' value in chatter mail.